content/en/blog/_posts/XXXX-XX-XX-fine-grained-supplementalgroups-control-ga.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: blog
+title: 'Kubernetes v1.35: Fine-grained SupplementalGroups Control Graduates to GA'
+date: XXXX-XX-XX
+draft: true
+slug: kubernetes-v1-35-fine-grained-supplementalgroups-control-ga
+author: >
+  Shingo Omura (LY Corporation)
+
+---
+
+Announcing graduation to General Availability (GA) - Fine-grained SupplementalGroups Control in Kubernetes v1.35!
+
+The new field, `supplementalGroupsPolicy`, was introduced as an opt-in alpha feature for Kubernetes v1.31, and then had graduated to beta in v1.33; Now, the corresponding feature gate (`SupplementalGroupsPolicy`) is now GA. This feature enables to implement more precise control over supplemental groups in containers that can strengthen the security posture, particularly in accessing volumes. Moreover, it also enhances the transparency of UID/GID details in containers, offering improved security oversight.
+
+If you upgrade from v1.32 or before, please be aware that some behavioral breaking change introduced since beta (v1.33). See [The behavioral changes introduced in beta](https://kubernetes.io/blog/2025/05/06/kubernetes-v1-33-fine-grained-supplementalgroups-control-beta/#the-behavioral-changes-introduced-in-beta) and [Upgrade Considerations](https://kubernetes.io/blog/2025/05/06/kubernetes-v1-33-fine-grained-supplementalgroups-control-beta/#upgrade-consideration) sections in [the previous blog for beta release](https://kubernetes.io/blog/2025/05/06/kubernetes-v1-33-fine-grained-supplementalgroups-control-beta/) for details.
```

Comment:
```suggestion
If you upgrade from v1.32 or earlier version, please be aware that some behavioral breaking change introduced since beta (v1.33). See [The behavioral changes introduced in beta](https://kubernetes.io/blog/2025/05/06/kubernetes-v1-33-fine-grained-supplementalgroups-control-beta/#the-behavioral-changes-introduced-in-beta) and [Upgrade Considerations](https://kubernetes.io/blog/2025/05/06/kubernetes-v1-33-fine-grained-supplementalgroups-control-beta/#upgrade-consideration) sections in [the previous blog for beta release](https://kubernetes.io/blog/2025/05/06/kubernetes-v1-33-fine-grained-supplementalgroups-control-beta/) for details.
```